rubrics/services/pom.xml

```diff
@@ -25,56 +25,83 @@
         <rest.assured.version>2.9.0</rest.assured.version>
         <spring.security.version>4.2.1.RELEASE</spring.security.version>
         <spring.boot.version>1.5.2.RELEASE</spring.boot.version>
```

Comment:
Do  you think it's worth pushing putting more of these properties up into master/depManagement here? Or just leaving them since this is the only project that's really using them right now.